core/include/crypto/crypto.h

```diff
@@ -83,6 +83,22 @@ void crypto_authenc_copy_state(void *dst_ctx, void *src_ctx);
 
 #if defined(CFG_NXP_SE05X)
 TEE_Result crypto_enable_scp03(unsigned int rotate_keys);
+
+enum crypto_apdu_type {
+	CRYPTO_APDU_NONE,
+	CRYPTO_APDU_TYPE_0,
+	CRYPTO_APDU_TYPE_1,
+	CRYPTO_APDU_TYPE_2,
+	CRYPTO_APDU_TYPE_3,
+	CRYPTO_APDU_TYPE_4,
+	CRYPTO_APDU_TYPE_5,
+	CRYPTO_APDU_TYPE_6,
```

Comment:
they are just a commodity flag to simplify the actual functions that will handle the transfers...using them, since I am integrating with a generic NXP service, is simpler and will require less maintenance than modifying the service.